UnwrapTests/BundleLoadingTests.swift

```diff
@@ -29,7 +29,7 @@ class BundleLoadingTests: XCTestCase {
     /// Tests that strings can be formatted in a bundle-friendly way.
     func testStringBundleNames() {
         let testString = "This?is , a:test'"
-        XCTAssert(testString.bundleName == "Thisis--atest")
+        XCTAssert(testString.bundleName == "thisis--atest")
```

Comment:
Hey, thanks for fixing the broken tests! Since we're here, wouldn't it be another win to use `XCTAssertEqual` instead of `XCTAssert` to get better test failure messages? Otherwise, we'll get the message "XCTAssertTrue failed" for failures, which is not very helpful.

`XCTAssertEqual(testString.bundleName, "thisis--atest")` 

Error example: `XCTAssertEqual failed: ("thisis--atest") is not equal to ("Thisis--atest")`

Reply:
yup! totally make sense 👍